src/PointwiseFunctions/AnalyticData/GrMhd/OrszagTangVortex.hpp

```diff
@@ -0,0 +1,170 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+// IWYU pragma: no_include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"  // for IdealFluid
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace AnalyticData {
+
+/*!
+ * \brief Analytic initial data for the relativistic Orszag-Tang vortex.
+ *
+ * The relativistic version of the Orszag-Tang vortex, described, for
+ * example, in [arXiv:1101.3573](https://arxiv.org/abs/1101.3573), is
+ * a 2-dimensional test case for relativistic MHD systems.  We do not
+ * currently support 2-dimensional RMHD, so this class provides
+ * 3-dimensional data with no \f$z\f$-dependence.
+ *
+ * The problem describes the flow of an ideal fluid with adiabatic
+ * index \f$5/3\f$.  The initial conditions (and hence the states at
```

Comment:
I see. I think the note on the dimensionality makes this structure a bit unclear, because it is a detail and so doesn't fit in the introduction. I suggest making everything one paragraph, followed by the note on dimensionality:

> The relativistic version of the Orszag-Tang vortex is a 2-dimensional test case for relativistic MHD systems \cite TheRefKey. It describes the flow of an ideal fluid with adiabatic index \f$\gamma=\frac{5}{3}\f$. The initial conditions are ...
>
> \note We do not currently support ...

You could also add some more details on what makes this analytic data useful (shock-shock interactions, divergence cleaning)

Read this as a suggestion and feel free to change.